pontoon/base/templates/widgets/filter_item.html

```diff
@@ -1,9 +1,16 @@
-{% macro item(type, title, count=False) %}
+{% macro item(type, title, count=False, priority=None) %}
 <li class="{{ type }}" data-type="{{ type }}">
   <span class="status fa"></span>
   <span class="title">{{ title }}</span>
   {% if count %}
-    <span class="count"></span>
+  <span class="count"></span>
+  {% endif %}
+  {% if priority %}
```

Comment:
Currently, template gets only tags with set priority and because of that this condition will be always true.

Reply:
It actually gets all the tags. Priority is an optional field.